master/getting-started/bare-metal/bare-metal-install.md

```diff
@@ -6,15 +6,9 @@ These instructions will take you through a first-time install of
 Calico's per-host daemon, Felix, starting with the calico-felix binary.
 These instructions apply to Calico v2.1.0 and above.  Older versions
  were packaged differently.
+ 
+<div class="alert alert-info" role="alert"><b>Note</b>: This install process is most suited to bare-metal-only installations where Felix is to be used to control policy for the host's interfaces. For OpenStack and containers there are additional daemons that need to be installed, which are not covered here.</div>
```

Comment:
Is it possible for me to see an example of what this new format looks like?

Reply:
Also, in general, could the changes in this PR be made more minimal and easier to review by only:

- changing `> ** NOTE**` to `<div class="alert alert-info" role="alert"><b>Note</b>:`

- removing the leading `> ` on other lines of the note content

- adding `</div>` after the end of the note content.

That would make it easier to see that the note content is not changing, and would also preserve the existing line breaks (which some of us find helpful).

Reply:
@neiljerram I see your point about adjusting the edits so that the reviewers can more clearly see what changed inside of the notes themselves. I will try to do this, but it could take some time, that's all. 

I realize that the new notes do not look so beautiful from a code perspective, but they look a lot better on the website itself. We are trying to optimize for the experience of people visiting the website versus people visiting the source code.

You can easily view how this looks by building the site locally via `make serve` as discussed in CONTRIBUTING_DOCS.md.